PSAutobahnClientTests/PSAutobahnClientTests.m

```diff
@@ -156,12 +156,19 @@ - (void)recordFailureWithDescription:(NSString *)description inFile:(NSString *)
 - (void)runOperation:(NSOperation *)operation timeout:(NSTimeInterval)timeout {
     static NSOperationQueue *queue = nil;
     static dispatch_once_t once = 0;
-    dispatch_once(&once, ^{
-        queue = [[NSOperationQueue alloc] init];
-        queue.maxConcurrentOperationCount = NSOperationQueueDefaultMaxConcurrentOperationCount;
-    });
-    
-    
+    @try {
+        dispatch_once(&once, ^{
+            queue = [[NSOperationQueue alloc] init];
+            queue.maxConcurrentOperationCount = NSOperationQueueDefaultMaxConcurrentOperationCount;
+        });
+    }
+    @catch (NSException *exception) {
+        NSLog(@"%@", exception.reason);
+        NSLog(@"Crash when runOperation");
+    }
+    @finally {
+        NSLog(@"Finally condition");
```

Comment:
1. Suggestion: the finally log can be more descriptive, such as `runOperation finished`
2. I think this is just a test file, we don't have to add try catch here?

Reply:
1. Agree!
2. Yes, I didn't notice that this is a test file, thank you Jesse!